SQL/2016-06-27-VideoUploader.sql

```diff
@@ -0,0 +1,52 @@
+
+-- Add 'videos' tab to the menu under Clinical section
+DELETE FROM LorisMenu WHERE Label='Videos';
+INSERT INTO LorisMenu (Label, Link, Parent, OrderNumber) VALUES ('Videos', 'videos/', (SELECT ID FROM LorisMenu as L WHERE Label='Clinical'), 5);
+
+-- Add 'videos' table
+DROP TABLE IF EXISTS `videos`;
+CREATE TABLE `videos` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `pscid` varchar(255) NOT NULL DEFAULT '',
+  `visit_label` varchar(255) NOT NULL DEFAULT '',
+  `instrument` varchar(255) DEFAULT NULL,
+  `for_site` int(2) DEFAULT NULL,
+  `date_taken` date DEFAULT NULL,
+  `comments` text,
+  `file_name` varchar(255) DEFAULT NULL,
+  `file_type` varchar(255) DEFAULT NULL,
+  `file_size` bigint(20) unsigned DEFAULT NULL,
+  `data_dir` varchar(255) DEFAULT NULL,
+  `uploaded_by` varchar(255) DEFAULT NULL,
+  `hide_video` tinyint(1) DEFAULT '0',
+  `date_uploaded` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`)
+);
+
+
+-- Add user permissions
+DELETE FROM permissions WHERE code='video_upload';
+INSERT INTO permissions (`code`, `description`, `categoryID`) VALUES (
+	'video_upload', 'Video uploading ', 1
+);
```

Comment:
@driusan I don't quite understand why is it a problem that permID would change for video_upload permission, since this script updates other tables that use it as a foreign key. Can you clarify what exactly would it break?

Also, the only solution that I found to rewrite this without changing ID or modifying the structure of permissions table is:
```
SET @currentPermissionID = (SELECT permID FROM permissions WHERE code='video_upload');
INSERT IGNORE INTO permissions (`permID`, `code`, `description`, `categoryID`) VALUES (
	@currentPermissionID, 'video_upload', 'Video uploading ', 1
);
```
Do you consider that a viable solution or there's is something better I can do?